### PR TITLE
Review base modeling

### PR DESCRIPTION
**Pull request by mnsnor**
Har inte kollat om den compilar i vårt LaTex frameworks, men då är det ju bara att lägga till rätt \userpackage

### CODE REVIEW

**Comment by edwced** [comment]
I'll look it over on friday 11/12.

File: report/finished/chapters/modeling.tex [hunk 1]
```diff
@@ -1,10 +1,60 @@
 \section*{Robotic arm}
 \subsection*{Forward kinematics}
 \subsection*{Inverse kinematics}
+
+
+%%%%%%%%%%%%%%%%%%
+
 \section*{Base}
 
+ The motors will be connected to tracks on the robot and can therefore be modelled as two wheels connected with a rod, as seen in figure \ref{fig:base_math_model}.\\
```
Comment (Partins) [inline report/finished/chapters/modeling.tex]: figure should be changed to "Fig. \ref{}". AlsoI think "SHOWN in Fig..." is better than seen

File: report/finished/chapters/modeling.tex [hunk 2]
```diff
@@ -1,10 +1,60 @@
...
 \subsection*{Inverse kinematics}
+
+
+%%%%%%%%%%%%%%%%%%
+
 \section*{Base}
 
+ The motors will be connected to tracks on the robot and can therefore be modelled as two wheels connected with a rod, as seen in figure \ref{fig:base_math_model}.\\ 
+This means that most of the mathematical modeling is very straight forward given that we can read the encoders from the motors. From the encoders we would be able to get both the length the robot has traveled and more importantly the individual speed each motor rotates with. The speed for the individual motor is of course given by: 
+
```
Comment (Partins) [inline report/finished/chapters/modeling.tex:12]: "of course" is not necessary.

File: report/finished/chapters/modeling.tex [hunk 3]
```diff
@@ -1,10 +1,60 @@
...
+\end{figure}
+
+Given small angles \footnote{Which will be assumed because the robot will be operating on a grid with hard coded functions for turning at QR-codes}, we can therefore write this as:
+
+\begin{equation}
+    \theta = \frac{V_L-V_R}{2r_{base}}t
+\end{equation}
+
+
+\noindent Using equations \eqref{eq:base_system_eq1}-\eqref{eq:base_system_eq2} a system can be built for simulations in matlab, using rate of change and max/min value limiters for simulations of the motors physical restrictions.
```
Comment (Partins) [inline report/finished/chapters/modeling.tex:51]: should be MATLAB, I even think it's a registered trademark

File: report/finished/chapters/modeling.tex [hunk 4]
```diff
@@ -1,10 +1,60 @@
...
 \subsection*{Forward kinematics}
 \subsection*{Inverse kinematics}
+
+
+%%%%%%%%%%%%%%%%%%
+
 \section*{Base}
 
+ The motors will be connected to tracks on the robot and can therefore be modelled as two wheels connected with a rod, as seen in figure \ref{fig:base_math_model}.\\ 
+This means that most of the mathematical modeling is very straight forward given that we can read the encoders from the motors. From the encoders we would be able to get both the length the robot has traveled and more importantly the individual speed each motor rotates with. The speed for the individual motor is of course given by:
```
Comment (edwced) [inline report/finished/chapters/modeling.tex]: line 11:
Deriving the mathematical model is straight forward given.... 
* should thin out words that acts as "fillers"

The speed for the individual motor is  given by: 
*remove of course, "fillers"

File: report/finished/chapters/modeling.tex (hunk 2 above)
Comment (edwced) [inline report/finished/chapters/modeling.tex:12]: might be a consideration to do shorter subs in the equation like r_{wheel} = r_{wh} etc

File: report/finished/chapters/modeling.tex [hunk 5]
```diff
@@ -1,10 +1,60 @@
...
+    \label{eq:base_system_eq1}
+\end{equation}
+
+\noindent Where $N_{encoders}$ is the number of encoders on the motor, $r_{wheel}$ is the radius of the driving wheel and thickness of the track and $\Delta t$ is the time between the previous encoder reading and the most recent one. By doing this for both the left and right motor we can find how fast the robot is moving along the line by:
+
+\begin{equation}
+    \overline{v}= \frac{v_L+v_R}{2}(-\cos \theta \hat{i}+ \sin \theta \Hat{j})
+\end{equation}
+
+\noindent Where the $y$-axis parallel to the line. In a similar fashion to how we calculated the velocity of the robot, the rate of change of the angle can be calculated using:
```
Comment (edwced) [inline report/finished/chapters/modeling.tex]: angular velocity?

File: report/finished/chapters/modeling.tex [hunk 6]
```diff
@@ -1,10 +1,60 @@
...
+\end{equation}
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.7\textwidth]{base_math_model.eps}
+    \caption{Modelling of the base as two wheels connected with a rod, where $\theta$ is the angle to the line}
+    \label{fig:base_math_model}
+\end{figure}
+
+Given small angles \footnote{Which will be assumed because the robot will be operating on a grid with hard coded functions for turning at QR-codes}, we can therefore write this as:
```
Comment (edwced) [inline report/finished/chapters/modeling.tex]: small-angle approximation* Might need a reference in the footnotes.
(might be self-explained, so i guess).

File: report/finished/chapters/modeling.tex [hunk 7]
```diff
@@ -1,10 +1,60 @@
 \section*{Robotic arm}
 \subsection*{Forward kinematics}
 \subsection*{Inverse kinematics}
+
+
+%%%%%%%%%%%%%%%%%%
+
 \section*{Base}
```
Comment (edwced) [inline report/finished/chapters/modeling.tex:9]: The equation references dont work for me when i compile, also the picture dont load. I've sync with REVIEW_BaseModeling.

**Review by edwced — APPROVED** [review]
Good changes!

**Review by Partins — APPROVED** [review]
nice!

**Pull request merged**